packages/studio-base/src/components/OpenDialog/Start.tsx

```diff
@@ -64,15 +64,13 @@ export default function Start(props: IStartProps): JSX.Element {
         iconProps: { iconName: "Flow" },
         onClick: () => onSelectView("connection"),
       },
-      /*
-    {
-      id: "demo-data",
-      children: "Explore sample data",
-      secondaryText: "New to Studio? Have a play with demo data",
-      iconProps: { iconName: "BookStar" },
-      onClick: () => onSelectView("demo"),
-    },
-    */
+      {
+        id: "demo-data",
+        children: "Explore sample data",
+        secondaryText: "New to Studio? Have a play with demo data",
```

Comment:
Maybe this is just an American thing but "have a play" sounds unusual to me.
```suggestion
        secondaryText: "New to Studio? Try out the app with some demo data",
```

Reply:
`Try some demo data`?

Try our the app somehow sounds verbose... anyhow - can make this whatever we think is good language.

Reply:
Updated the language.